blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -189,8 +189,10 @@ export default async function decorate(block) {
     buildProductCard(program);
     try {
         const imageObject = await searchAsset(program.programName, program.campaignName);
-        insertImageIntoCampaignImg(block,imageObject);
-        document.getElementById('totalassets').textContent = imageObject.assetCount;
+        if (imageObject){
+          insertImageIntoCampaignImg(block,imageObject);
+          document.getElementById('totalassets').textContent = imageObject.assetCount;
```

Comment:
@TyroneAEM can you return total asset count to zero. In UI, it does not display any number.
![image](https://github.com/hlxsites/adobe-gmo/assets/61603050/f6412299-b7bb-44eb-9d61-7c57a8443b5b)
![image](https://github.com/hlxsites/adobe-gmo/assets/61603050/b4d17964-9a01-48f2-8379-548042795fad)